crates/proposer-client/Cargo.toml

```diff
@@ -26,16 +26,28 @@ tonic.workspace = true
 tracing.workspace = true
 url.workspace = true
 
+# Ecosystem dependencies
+op-succinct-grpc = { git = "https://github.com/agglayer/op-succinct.git", branch = "vcastellm/rust-agglayer", package = "grpc" }
```

Comment:
Note: This pulls in the generated gRPC types as a dependency instead of copying the proto file and generating the code locally (in this repo). For now, a branch is specified rather than a version or a tag. As far as I can tell, there are no tags or releases of the `rust-agglayer` branch at the moment.